src/wallet/wallet_rpc_server.cpp

```diff
@@ -4502,10 +4502,12 @@ class t_daemon
       const auto arg_wallet_file = wallet_args::arg_wallet_file();
       const auto arg_from_json = wallet_args::arg_generate_from_json();
       const auto arg_rpc_client_secret_key = wallet_args::arg_rpc_client_secret_key();
+	  const auto arg_password_file = wallet_args::arg_password_file();
 
       const auto wallet_file = command_line::get_arg(vm, arg_wallet_file);
       const auto from_json = command_line::get_arg(vm, arg_from_json);
       const auto wallet_dir = command_line::get_arg(vm, arg_wallet_dir);
+	  const auto password_file = command_line::get_arg(vm, arg_password_file);
```

Comment:
You've just added tabs, which screws up indentation above. Please only add tabs to code that already uses tabs.

Reply:
Sorry about that - fixed.